src/component/Sticky/Sticky.test.tsx

```diff
@@ -0,0 +1,135 @@
+import { Sticky, STICKY_SPEC_ID, StickyProps } from './Sticky';
+import { createSetup, createSpy, getSpecWrapper } from '../../testUtils';
+import { FlowDefinition, StickyNote } from '../../flowTypes';
+import { getBaseLanguage, dump } from '../../utils';
+import { ConnectionEvent, createStore, initialState } from '../../store';
+import { object } from 'prop-types';
+import { endpointsPT, languagesPT } from '../../config/ConfigProvider';
+import * as styles from './Sticky.scss';
+import * as config from '../../../assets/config';
+
+jest.useFakeTimers();
+
+// TODO: this is an awful lot of boilerplate for testing a simple component
```

Comment:
this doesn't feel _too_ bad to me (makes sense to be explicit about prop requirements, context subscriptions). wonder if having `context` and `childContextTypes` initialized by default in `createSetup` wouldn't help? that's the most laborious part for me, anyways, and there's no harm in passing all of `context` to components that don't read from it.

Reply:
the drier the better as far as im concerned. need to keep the tests as concise and just meat as possible.

Reply:
ok probably worth opening a PR for that. i can make that happen when this is merged, unless you want to tack that on.